vuu-ui/packages/vuu-utils/src/range-utils.ts

```diff
@@ -12,12 +12,21 @@ interface FromToRange {
 
 export const NULL_RANGE: VuuRange = { from: 0, to: 0 } as const;
 
+export const rangesAreSame = (
+  r1: VuuRange | undefined,
+  r2: VuuRange | undefined
+) => {
+  return r1?.from === r2?.from && r1?.to === r2?.to;
+};
+
 export function getFullRange(
   { from, to }: VuuRange,
   bufferSize = 0,
   rowCount: number = Number.MAX_SAFE_INTEGER
```

Comment:
Does this represent total row count? If yes, should we rename it to `totalRowCount` for better readability? Wdyt?

Reply:
agreed. Have renamed it. As you will have seen, rowCount sometimes refers to number of rows in viewport. Good to be explicit.